lmdeploy/serve/openai/protocol.py

```diff
@@ -191,6 +191,19 @@ class EmbeddingsResponse(BaseModel):
     usage: UsageInfo
 
 
+class EncodeRequest(BaseModel):
+    """Encode request."""
+    input: Union[str, List[str]]
+    do_preprocess: Optional[bool] = False
+    add_bos: Optional[bool] = True
+
+
+class EncodeResponse(BaseModel):
+    """Encode response."""
+    input_ids: Union[List[int], List[List[int]]]
+    length: Union[int, List[int]]
```

Comment:
Since `input_ids` a list is returned, is `length` necessary?

Reply:
it's just for opencompass. I did not find any other usage.